src/components/Image/index.tsx

```diff
@@ -53,21 +47,19 @@ function Image(props) {
     );
 }
 
-function imagePropsAreEqual(prevProps, nextProps) {
+function imagePropsAreEqual(prevProps: ImageOwnProps, nextProps: ImageOwnProps) {
     return prevProps.source === nextProps.source;
```

Comment:
This is unrelated to this PR, but the Image will not re-render if styles or other props change.

@kowczarz, Could you share your insights on comparing only the source in the `imagePropsAreEqual` function?